CHANGES.md

```diff
@@ -6,7 +6,40 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
-## Trino Python client 0.316.0
+## Release 0.317.0
+
+* Add support for creating tables containing `JSON` columns and reading and
+  writing to them with SQLAlchemy.
+  ([#194](https://trinodb/trino-python-client/issues/194))
+* Add support for setting roles by passing a dictionary of catalog name and
+  role as the `roles` keyword argument to `trino.dbapi.connect`.
+  ([#230](https://trinodb/trino-python-client/issues/230))
+* Add support for setting roles by adding the `roles` URL query parameter in
+  SQLAlchemy connections to a JSON object with keys as the catalog name and
+  values as the role name.
+  ([#230](https://trinodb/trino-python-client/issues/230))
+* Add a function `trino.sqlalchemy.URL` to generate SQLAlchemy URLs which
+  properly handles escaping and encoding values where needed.
+  ([#235](https://trinodb/trino-python-client/issues/235))
+* Fix query failures not being propagated to the client when using `fetchone`.
+  ([#95](https://trinodb/trino-python-client/issues/95))
+* Fix queries returning a single row from sometimes appearing as failed on the
+  server. ([#220](https://trinodb/trino-python-client/issues/220))
+* Fix query failures when using SQLAlchemy `TableClause` by not performing
+  catalog lookup. ([#237](https://trinodb/trino-python-client/issues/237))
+* Fix errors when using prepared statements with Trino versions greater than or
+  equal to 398. ([#242](https://trinodb/trino-python-client/issues/242))
+
+### Breaking Changes
+
+* Block the `execute` method of the cursor until at least one row is received.
+  Users no longer need to call `fetchone` or `fetchall` to ensure query starts
+  executing on the Trino server. Note that results still need to be consumed by
+  calling `fetchone` or `fetchall` to ensure that a query isn't considered idle
+  and terminated on the server.
+  ([#232](https://trinodb/trino-python-client/issues/232))
```

Comment:
Could you change all URL to contain `github.com`? 
`https://trinodb/trino-python-client/issues/232` → `https://github.com/trinodb/trino-python-client/issues/232`

Reply:
Thanks for catching this. 🤦